elastic-search-support/src/main/resources/globalSearchCasesMapping.json

```
@@ -56,6 +56,18 @@
       "analyzer": "standard",
       "search_analyzer": "alphanumeric_string_analyzer"
     },
+    "case_access_category": {
```

```suggestion
    "CaseAccessCategory": {
```
A couple of points:
1. The field id should be `CaseAccessCategory`
1. It is a top level case data field so should be within the `data` object (alongside `SearchCriteria`, `caseManagementLocation` etc.)

---

DONE